static/src/js/components/Facets/FacetsItem.jsx

```diff
@@ -145,7 +145,7 @@ class FacetsItem extends Component {
               )
             }
           </div>
-          { (!applied || !children) && facet.count && <span className="facets-item__total">{commafy(facet.count)}</span> }
+          { (!applied || !children) && (facet.count !== undefined) && <span className="facets-item__total">{commafy(facet.count)}</span> }
```

Comment:
```suggestion
          { (!applied || !children) && (facet.count facet.count != null) && <span className="facets-item__total">{commafy(facet.count)}</span> }
```
The current change does fix the edge case which should not happen in the first place but, is a state we can currently enter due to a bug on the API. This suggested solution above would cover if the `facet.count` is `undefined` or `null` by loose equality